docker-compose-rule-core/src/main/java/com/palantir/docker/compose/configuration/ShutdownStrategy.java

```diff
@@ -55,6 +38,8 @@
      */
     ShutdownStrategy SKIP = new SkipShutdownStrategy();
 
-    void shutdown(DockerCompose dockerCompose, Docker docker) throws IOException, InterruptedException;
+    void stop(DockerCompose dockerCompose) throws IOException, InterruptedException;
+
+    void shutdown(DockerCompose dockerCompose) throws IOException, InterruptedException;
```

Comment:
I think if you're going to turn a one-method interface into a two method interface then it's really worth providing some javadoc to describe the semantics of these... especially as `shutdown` is a DCR concept, not a vanilla docker one

Reply:
The concepts here align with the docker concepts of `stop` and `rm`. docker-compose `down` is just `stop` + `rm`. I'll update the method names to make this more clear.